### PR TITLE
Decoupling Error whose patter is InvalidEntityException "The name is invalid"

### PR DESCRIPTION
**Pull request by jmmovilla**
@jframos @jesuspg

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/exception/InvalidOVFException.java [hunk 1]
```diff
@@ -22,18 +22,18 @@
...
+public class InvalidOVFException extends Exception {
+
+    public InvalidOVFException(String msg) {
+        super(msg);
+    }
+
+    public InvalidOVFException(Throwable e) {
+        super(e);
+    }
+}
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/exception/InvalidOVFException.java]: Just add a new line...

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/InfrastructureManagerClaudiaImpl.java [hunk 2]
```diff
@@ -139,7 +139,7 @@ public EnvironmentInstance createInfrasctuctureEnvironmentInstance(EnvironmentIn
                 log.info("Number of networks " + tierInstance.getNetworkInstances().size() + " floatin ip "
                         + tierInstance.getTier().getFloatingip());
 
-                try {
+                //try {
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/InfrastructureManagerClaudiaImpl.java]: If you are using a version control system, do you need to put in the commit this commented lines?

**Comment by flopezag** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/exception/InvalidOVFException.java [hunk 3]
```diff
@@ -22,18 +22,18 @@
...
-    }
-
-    public InvalidOVFException(Throwable e) {
-        super(e);
-    }
-}
+package com.telefonica.euro_iaas.paasmanager.exception;
+
+/**
+ * @author jesus.movilla
```
Comment (jesuspg) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/exception/InvalidOVFException.java]: I recommend don't write label @author

**Comment by jframos** [comment]
LGTM
(To be tested in Experimentation Environment)

**Pull request merged**